SPECS/grub2/50_mariner

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+cat << EOF
+set rootdevice=$GRUB_MARINER_PARTUUID
+load_env -f $GRUB_MARINER_BOOTPREFIX/mariner.cfg
+
+menuentry "CBL-Mariner" {
+	linux $GRUB_MARINER_BOOTPREFIX/\$mariner_linux $GRUB_CMDLINE_LINUX
+	if [ -f $GRUB_MARINER_BOOTPREFIX/\$mariner_initrd ]; then
+		initrd $GRUB_MARINER_BOOTPREFIX/\$mariner_initrd
```

Comment:
this is the wrong way to do it; we need to drop the mariner.cfg file completely and stop using $mariner_* vars, instead this should generate the grub.cfg with the actual values directly